src/appleseed/renderer/modeling/bssrdf/randomwalkbssrdf.cpp

```diff
@@ -495,7 +495,7 @@ namespace
             assert(abs(cosine) <= 1.0f);
```

Comment:
`std::abs()`

Reply:
I thought we had an `abs` in `foundation::`, but rereading the foundation code makes me think it's not meant to replace `std::abs`, will include it too.